### DISCUSSION TITLE
ESP32-3248S035C Support

### DISCUSSION THREAD

**Discussion by kyledama** [Bodmer/TFT_eSPI#3122] [Compatible displays and setup files]
Hi,

I'm wondering if this library support the ST7796 ESP32-3248S035_C?

Here's the one I ordered - https://www.amazon.ca/dp/B0C4KSKW96?psc=1&ref=ppx_yo2ov_dt_b_product_details

I believe its the same one as this - https://www.openhasp.com/0.7.0/hardware/sunton/esp32-3248s035/

Thanks in advance,

Kyle

**Comment by Bodmer** [comment]
The library supports the ESP32 and ST7796 display with 4 wire SPI and 8 bit parallel interfaces.

**Reply by kyledama** [reply]
Do you have any recommendations for a 3"+ display that is compatible and works well?
